Description: Fix *.xsl files
 Remove a missing/unneeded import on jargon.xsl
 Disable the ToC generation on jargon-text.xsl
Origin: Debian jargon-text package
Forwarded: no
Reviewed-By: Ruben Molina <ruben.molina@udea.edu.co>
Last-Update: 2015-09-03

--- dict-jargon-4.4.7.orig.orig/jargon-text.xsl
+++ dict-jargon-4.4.7.orig/jargon-text.xsl
@@ -6,8 +6,8 @@
 
 <xsl:import href="jargon.xsl"/>
 
-<xsl:param name="suppress.glossary.titlepage" select="1"/>
-<xsl:param name="suppress.glossdev.contents" select="1"/>
+<xsl:param name="suppress.glossary.titlepage" select="0"/>
+<xsl:param name="suppress.glossdev.contents" select="0"/>
 
 <!-- Surround links with {} as in older versions. -->
 <xsl:template match="glossdef//glossterm">
--- dict-jargon-4.4.7.orig.orig/jargon.xsl
+++ dict-jargon-4.4.7.orig/jargon.xsl
@@ -2,7 +2,6 @@
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
                 version="1.0">
 
-<xsl:import href="docbook-xsl-1.61.0/xhtml/docbook.xsl"/>
 
 <xsl:param name="navig.graphics" select="0"/>
 <xsl:param name="navig.graphics.path">../graphics/</xsl:param>
